### DISCUSSION TITLE
Notable contributions: filter out specific organizations/repos?

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#764] [Q&A]
I don't think the option to filter out an individual organization/repo (regardless of star count, forks, etc.) exists yet.
https://github.com/lowlighter/metrics/blob/b1004bf9ac3038ea5c973f4d5a20f0d6e3e46322/source/plugins/notable/index.mjs#L23

I don't know if this is representative of metrics's broader audience, but there is a notable contribution that I actually want excluded. The reason is that it wasn't a *good* contribution (introduced bug), resulting in most of my contribution being undone :facepalm: 
It would be unfair to say that I "contributed" to that organization :laughing:

**Comment by lowlighter** [comment]
I'm not sure since I have not tested myself, but `user: NOT filtered-out` should work

The syntax is supposed to feel like [GitHub search syntax](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#exclude-certain-results), but since it's an homemade implementation it may not follow the whole spec (the `-` notation for removal isn't implemented iirc for example)

**Reply by spenserblack** [reply]
Ok, I'll mark change this to Q&A since it might be supported after all.

It looks like `NOT` *must* be followed by a search term, not a qualifier. E.g. If my search is `user:lowlighter NOT repo:lowlighter/metrics` I get no results. So I'm not sure what the best way to exclude a repo is. Currently trying to test searches in GitHub itself before trying to test it out with the metrics workflow.

**Reply by lowlighter** [reply]
yes indeed it might be the contrary

**Reply by spenserblack** [reply]
Actually, could Notable Contributions inherit the base plugin's `repositories_skipped` option? Besides meeting my need, I think that it would be expected behavior for users if repositories listed in `repositories_skipped` wouldn't appear in the Notable Contributions plugin.

**Reply by lowlighter** [reply]
Yeah it's a good idea

**Reply by lowlighter** [reply]
The inheritance for `repositories_skipped` has been implemented in #789 👍
